scripts/normalize-by-median.py

```diff
@@ -55,11 +53,17 @@ def WithDiagnostics(ifile, batch_size, fp, paired, norm):
 
     index = 0
 
-    for index, batch in enumerate(batchwise(
-                                  screed.open(ifile, parse_description=False),
-                                  batch_size)):
+    screed_iter = screed.open(ifile, parse_description=False)
+    reader = broken_paired_reader(screed_iter, force_single=single)
```

Comment:
reader should be passed in; broken_paired_reader should be called outside of WithDiagnostics; filename can be passed in together with reader but should only be used for reporting.